core/src/main/java/com/linecorp/armeria/common/RequestId.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common;
+
+/**
+ * An identifier of a {@link Request}-{@link Response} pair.
+ * Note that this identifier does not guarantee uniqueness. A different {@link Request} may have the same
+ * {@link RequestId}, although its chance would be fairly low.
+ */
+public interface RequestId {
+
+    /**
+     * Returns a {@link RequestId} created from a 64-bit random integer.
+     */
+    static RequestId random() {
+        return new DefaultRequestId();
+    }
+
+    /**
+     * Returns the long textual representation of this ID.
```

Comment:
I think it can just be `text` and `shortText` since the long one should basically be a full representation.

Reply:
Yeah, renamed.